### PR TITLE
Implemented Chinese translations and refactored code as per standards

### PR DESCRIPTION
**Pull request by veeranjaneyuludavuluri**

### CODE REVIEW

File: src/app/auth/nv-login/nv-login.component.scss [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 .right-section {
     height: 100vh;
-    background-image: linear-gradient(160deg, $color-dark-blue-10, $color-dark-blue);
+    background-image: linear-gradient(160deg, $color-dark-blue-1F, $color-dark-blue);
 }
 
 .novatek-logo-center {
```
Comment (Sina-pz) [inline src/app/auth/nv-login/nv-login.component.scss:8]: change to nv-logo-center
empty?

File: src/app/auth/nv-login/nv-login.component.scss [hunk 2]
```diff
@@ -195,9 +181,9 @@ button {
 }
 
 ::ng-deep .alert-success {
-    background-color: #FCFFF5 !important;
+    background-color: $color-white-fc !important;
```
Comment (Sina-pz) [inline src/app/auth/nv-login/nv-login.component.scss:184]: why -fc?

File: src/app/auth/nv-login/nv-login.component.scss [hunk 3]
```diff
@@ -195,9 +181,9 @@ button {
...
     margin-bottom: 0px !important;
-    border-color: #A8C599 !important;
+    border-color:  $color-green-a8 !important;
```
Comment (Sina-pz) [inline src/app/auth/nv-login/nv-login.component.scss:186]: a8?

File: src/app/auth/nv-login/nv-login.component.scss [hunk 4]
```diff
@@ -207,33 +193,29 @@ button {
 }
 
 .sucess-Txt {
-    color: #1E561F;
+    color: $color-green-1e;
```
Comment (Sina-pz) [inline src/app/auth/nv-login/nv-login.component.scss:196]: same

File: src/app/auth/nv-login/nv-login.component.scss [hunk 5]
```diff
@@ -195,9 +181,9 @@ button {
...
 
 ::ng-deep .alert-success {
-    background-color: #FCFFF5 !important;
+    background-color: $color-white-fc !important;
     margin-bottom: 0px !important;
-    border-color: #A8C599 !important;
+    border-color:  $color-green-a8 !important;
 }
 
 .secondary-txt {
```
Comment (Sina-pz) [inline src/app/auth/nv-login/nv-login.component.scss:189]: ?

File: src/app/auth/nv-login/nv-login.component.scss [hunk 6]
```diff
@@ -207,33 +193,29 @@ button {
 }
 
 .sucess-Txt {
-    color: #1E561F;
+    color: $color-green-1e;
     font-size: 16px;
     font-family: 'Lato';
```
Comment (Sina-pz) [inline src/app/auth/nv-login/nv-login.component.scss:198]: use variable in base.scss

File: src/app/auth/nv-login/nv-login.component.scss [hunk 7]
```diff
@@ -207,33 +193,29 @@ button {
...
 
 .alert-danger {
-    color: #973937;
-    background-color: #FFF6F6;
-    border-color: #f5c6cb;
-    border: 1px solid #973937;
+    color: $color-red-97;
+    background-color: $color-white-ff;
+    border-color: $color-light-red-f5;
+    border: 1px solid $color-red-97;
```
Comment (Sina-pz) [inline src/app/auth/nv-login/nv-login.component.scss:217]: ??

File: src/app/auth/nv-login/nv-login.component.ts [hunk 8]
```diff
@@ -61,52 +54,51 @@ export class LoginComponent implements OnInit {
...
+      width: '200',
+      height: '50',
     },
-    Domain :[
-        'Doctor',
-        'Pharma',
-        'Nurse'
-    ]
-};
+  };
```
Comment (Sina-pz) [inline src/app/auth/nv-login/nv-login.component.ts:60]: if possible find better solution for setting style in component

File: src/app/auth/nv-login/nv-login.component.ts [hunk 9]
```diff
@@ -25,25 +23,20 @@ export class LoginComponent implements OnInit {
...
   showForgotPasswordTxt = false;
   showConfirmationTxt = false;
-  // dropDown Menu
-  dropdownItemsMenu = this.mockMenuDropdown().items;
-  textTrimNumberMenu = this.mockMenuDropdown().textTrimNumber;
-  selectedItemDefaultMenu = this.mockMenuDropdown().selectedItemDefault;
-  dropDownFieldTypeMenu = this.mockMenuDropdown().dropDownFieldType;
-  public variableList:any = {
+
+  public variableList: any = {
```
Comment (Sina-pz) [inline src/app/auth/nv-login/nv-login.component.ts:27]: if possible change the name to a descriptive name

File: src/app/main/components/dashboard/dashboard.component.html [hunk 10]
```diff
@@ -1,98 +1,123 @@
...
+<app-nv-progressive-bar
+  [color]="'sucess'"
+  [size]="SucessprogressiveBarValue"
+  [progressiveTxt]="sucessTxt"
+></app-nv-progressive-bar>
+<br />
+<hr />
+<h1 class="comments-header"><strong>warning Progressive Bar</strong></h1>
+<app-nv-progressive-bar
+  [color]="'warning'"
```
Comment (Sina-pz) [inline src/app/main/components/dashboard/dashboard.component.html:29]: please define a variable in .ts file and assign it here instead

File: src/app/main/components/dashboard/dashboard.component.html [hunk 11]
```diff
@@ -1,98 +1,123 @@
...
+  [size]="warningProgressivebarValue"
+  [progressiveTxt]="warningTxt"
+  [statusText]="textColor"
+></app-nv-progressive-bar>
+<br />
+<hr />
+<h1 class="comments-header"><strong>error Progressive Bar</strong></h1>
+<app-nv-progressive-bar
+  [color]="'error'"
```
Comment (Sina-pz) [inline src/app/main/components/dashboard/dashboard.component.html:38]: same

File: src/app/main/components/dashboard/dashboard.component.html (hunk 10 above)
Comment (Sina-pz) [inline src/app/main/components/dashboard/dashboard.component.html:29]: please define a variable in .ts file and assign it here instead

File: src/app/reusable/nv-button/nv-button.component.scss [hunk 12]
```diff
@@ -42,7 +42,7 @@
   }
 
   &.theme-color-Primary {
-    background-color: $color-dark-blue-10;
+    background-color: $color-dark-blue-1F;
```
Comment (Sina-pz) [inline src/app/reusable/nv-button/nv-button.component.scss:45]: why you change the number to -1F ??? in nv-button!!!

File: src/app/reusable/nv-comment-box/nv-comment-box.component.scss [hunk 13]
```diff
@@ -27,7 +27,7 @@
   }
 
   .btn-color {
-    background-color: $color-dark-blue-10 !important;
+    background-color: $color-dark-blue-1F !important;
```
Comment (Sina-pz) [inline src/app/reusable/nv-comment-box/nv-comment-box.component.scss:30]: same

File: src/app/reusable/nv-input/nv-input.component.scss [hunk 14]
```diff
@@ -1,7 +1,7 @@
 @import "./../../reusable/theme/base.scss";
 
 .form-control:focus {
-    border-color: #727272;
+    border-color: $color-dark-gray-72;
```
Comment (Sina-pz) [inline src/app/reusable/nv-input/nv-input.component.scss:4]: same

File: src/app/reusable/nv-input/nv-input.component.scss [hunk 15]
```diff
@@ -23,17 +23,17 @@
...
 ::ng-deep .is-valid {
-    border-color:#A8C599;
-    background-color: #FCFFF5;
+    border-color:$color-green-a8;
+    background-color: $color-white-fc;
 }
 
 .is-notValid {
-    color: #DB2828;
+    color: $color-red;
```
Comment (Sina-pz) [inline src/app/reusable/nv-input/nv-input.component.scss:31]: same

File: src/app/reusable/nv-logo/nv-logo.component.ts [hunk 16]
```diff
@@ -6,11 +6,11 @@ import { Component, OnInit, Input } from '@angular/core';
...
 @Component({
-  selector: 'app-novatek-logo',
-  templateUrl: './novatek-logo.component.html',
-  styleUrls: ['./novatek-logo.component.scss']
+  selector: 'app-nv-logo',
+  templateUrl: './nv-logo.component.html',
+  styleUrls: ['./nv-logo.component.scss']
 })
-export class NovatekLogoComponent implements OnInit {
+export class NVLogoComponent implements OnInit {
```
Comment (Sina-pz) [inline src/app/reusable/nv-logo/nv-logo.component.ts:13]: NvLogo...

File: src/app/reusable/nv-progressive-bar/nv-progressive-bar.component.scss [hunk 17]
```diff
@@ -16,11 +16,11 @@
 }
 
 .firstBlock {
-    background-color: $color-progressive-first-block !important;
+    background-color: $color-grey-ad !important;
```
Comment (Sina-pz) [inline src/app/reusable/nv-progressive-bar/nv-progressive-bar.component.scss:19]: same

File: src/app/reusable/nv-progressive-bar/nv-progressive-bar.component.scss [hunk 18]
```diff
@@ -16,11 +16,11 @@
...
 }
 
 .secondBlock {
-    background-color: $color-progressive-second-block !important;
+    background-color: $color-grey-88 !important;
```
Comment (Sina-pz) [inline src/app/reusable/nv-progressive-bar/nv-progressive-bar.component.scss:23]: same

File: src/app/reusable/nv-progressive-bar/nv-progressive-bar.component.ts [hunk 19]
```diff
@@ -1,11 +1,11 @@
...
 @Component({
-  selector: 'app-progressive-bar',
-  templateUrl: './progressive-bar.component.html',
-  styleUrls: ['./progressive-bar.component.scss']
+  selector: 'app-nv-progressive-bar',
+  templateUrl: './nv-progressive-bar.component.html',
+  styleUrls: ['./nv-progressive-bar.component.scss']
 })
-export class ProgressiveBarComponent implements OnInit {
+export class NVProgressiveBarComponent implements OnInit {
```
Comment (Sina-pz) [inline src/app/reusable/nv-progressive-bar/nv-progressive-bar.component.ts:8]: NvPro..

File: src/app/reusable/nv-rating/nv-rating.component.scss [hunk 20]
```diff
@@ -16,12 +16,12 @@
   
   .rating-list li:hover,
   .rating-list li:hover ~ li, .rating-list li.selected {
-    color: $color-gold-star;
+    color: $color-yellow-de;
```
Comment (Sina-pz) [inline src/app/reusable/nv-rating/nv-rating.component.scss:19]: same

File: src/app/reusable/nv-right-tab/nv-right-tab.component.ts [hunk 21]
```diff
@@ -0,0 +1,17 @@
+import { Component, Input, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-nv-right-tab',
+  templateUrl: './nv-right-tab.component.html',
+  styleUrls: ['./nv-right-tab.component.scss']
+})
+export class NVRightTabComponent implements OnInit {
+
+  @Input() tabs:any;
```
Comment (Sina-pz) [inline src/app/reusable/nv-right-tab/nv-right-tab.component.ts:10]: if possible create a data-model for tabs

File: src/app/reusable/nv-slider-toggle/nv-slider-toggle.component.scss [hunk 22]
```diff
@@ -37,7 +37,7 @@
     border: 1px solid $color-white-10 !important;
   }
   ::ng-deep .mat-slide-toggle.mat-checked:not(.mat-disabled) .mat-slide-toggle-bar {
-      background-color: $color-dark-blue-10 !important;
+      background-color: $color-dark-blue-1F !important;
```
Comment (Sina-pz) [inline src/app/reusable/nv-slider-toggle/nv-slider-toggle.component.scss:40]: same

File: src/app/reusable/nv-tabs/nv-tabs.component.ts [hunk 23]
```diff
@@ -2,11 +2,11 @@ import { Component, Input, OnInit } from '@angular/core';
...
 @Component({
-  selector: 'app-tabs',
-  templateUrl: './tabs.component.html',
-  styleUrls: ['./tabs.component.scss']
+  selector: 'app-nv-tabs',
+  templateUrl: './nv-tabs.component.html',
+  styleUrls: ['./nv-tabs.component.scss']
 })
-export class TabsComponent implements OnInit {
+export class NVTabsComponent implements OnInit {
```
Comment (Sina-pz) [inline src/app/reusable/nv-tabs/nv-tabs.component.ts:9]: NvT

File: src/assets/icons/icon.arrow.left.square.svg [hunk 24]
```diff
@@ -0,0 +1,2 @@
+<svg id="ico.panel.options" xmlns="http://www.w3.org/2000/svg" width="24" height="23.999" viewBox="0 0 24 23.999">  <path id="ico.panel.options-2" data-name="ico.panel.options" d="M11596-3508a4.009,4.009,0,0,1-4-4v-16a4,4,0,0,1,4-3.994H11612a4,4,0,0,1,4,3.994v16a4.006,4.006,0,0,1-4,4Zm-2-20.005v16a2,2,0,0,0,2,2H11612a2.008,2.008,0,0,0,2.008-2v-16a2.008,2.008,0,0,0-2.008-2H11596A2,2,0,0,0,11594-3528Zm9.293,12.385-5.66-5.516a.952.952,0,0,1,0-1.379,1.032,1.032,0,0,1,1.418,0l4.941,4.827,4.961-4.827a1.018,1.018,0,0,1,1.406,0,.952.952,0,0,1,0,1.379l-5.66,5.516a1,1,0,0,1-.7.287A1.025,1.025,0,0,1,11603.293-3515.62Z" transform="translate(-11592 3531.998)" fill="#5d5d5d"/>
```
Comment (Sina-pz) [inline src/assets/icons/icon.arrow.left.square.svg:1]: the name of file of svg does not follow the convention:
name-x-type.svg => left-square-arrow.icon.svg

**Review by Sina-pz — COMMENTED** [review]
Good job
but in terms of styles, the consistency  is the most important thing; if you do not like the rules we already set please set new rules and we follow. but after that every body needs to follow it

File: src/app/auth/nv-login/nv-login.component.scss (hunk 5 above)
Comment (veeranjaneyuludavuluri) [reply src/app/auth/nv-login/nv-login.component.scss:189]: As we discussed earlier, removed hexcode and using from base css file.

File: src/app/auth/nv-login/nv-login.component.scss (hunk 2 above)
Comment (veeranjaneyuludavuluri) [reply src/app/auth/nv-login/nv-login.component.scss:184]: As we discussed earlier, removed hexcode and using from base css file. Kept name as first two letters of hex code.

File: src/app/auth/nv-login/nv-login.component.scss (hunk 1 above)
Comment (veeranjaneyuludavuluri) [reply src/app/auth/nv-login/nv-login.component.scss:8]: Changed to nv-logo-center. Didn't get you what you are referring to empty ?

File: src/app/auth/nv-login/nv-login.component.ts (hunk 8 above)
Comment (veeranjaneyuludavuluri) [reply src/app/auth/nv-login/nv-login.component.ts:60]: Already, I suggested prettier-plugin, that's the better one.

File: src/app/auth/nv-login/nv-login.component.ts (hunk 9 above)
Comment (veeranjaneyuludavuluri) [reply src/app/auth/nv-login/nv-login.component.ts:27]: Even I too thought of it, but it is used more than 70 places.

File: src/app/auth/nv-login/nv-login.component.ts [hunk 25]
```diff
@@ -61,52 +54,51 @@ export class LoginComponent implements OnInit {
...
 
-public emptyVariableList ={
-  confirmEmailInput: '',
-  emailInput: '',
-  userNameInput: '',
-  noValidation: '',
-  confirmEmailValidation: '',
-  emailValidation: '',
-}
+  // dropDown Menu
```
[the remark this answers is not in the document]
Comment (veeranjaneyuludavuluri) [reply src/app/auth/nv-login/nv-login.component.ts:62]: Just name is mockDropDown, it has actual values, didn't you check the function properly, have a look at mockMenuDropdown function.

File: src/assets/icons/icon.arrow.left.square.svg (hunk 24 above)
Comment (veeranjaneyuludavuluri) [reply src/assets/icons/icon.arrow.left.square.svg:1]: Updated

File: src/app/auth/nv-login/nv-login.component.ts [hunk 26]
```diff
@@ -290,27 +277,26 @@ public emptyVariableList ={
...
   private transilateLang(key: string) {
-    this.translate.get(key).subscribe(res => {
-      this.variableList[key] = res
+    this.translate.get(key).subscribe((res) => {
+      this.variableList[key] = res;
     });
   }
 
   onDomainChange(item: string): void {
     console.log('selected item ' + +item);
```
[the remark this answers is not in the document]
Comment (veeranjaneyuludavuluri) [reply src/app/auth/nv-login/nv-login.component.ts:286]: Done

File: src/app/auth/nv-login/nv-login.component.ts [hunk 27]
```diff
@@ -290,27 +277,26 @@ public emptyVariableList ={
...
+      textTrimNumber: 2, //5
+      selectedItemDefault: this.variableList.domainTxt, //'Select Domain',
       dropDownFieldType: DropdownFieldType.Default,
     };
     return dropdownInputs;
   }
 
-  login(){
+  login() {
     console.log('TODO - login');
```
[the remark this answers is not in the document]
Comment (veeranjaneyuludavuluri) [reply src/app/auth/nv-login/nv-login.component.ts:300]: Done

**Pull request merged**